src/de/bxservice/viesvatvalidation/process/ViesVATValidator.java

```diff
@@ -164,10 +183,53 @@ private void validateResponseName(JsonObject jsonResponse) {
 			addLog("@Name@ = " + name);
 	}
 	
+	private void validateResponseAddress(JsonObject jsonResponse) {
+		if(!isUpdateAddress) {
+			return;
+		}
+		
+		String address = getAddressFromResponse(jsonResponse);
+		
+		String [] parts = address.split("\n");
+		
+		if(parts.length < 2) {
+			return;
+		}
+	
+		String part2 = parts[1].trim();
+		
+		String address1 = address.split("\n")[0].trim();
+		String postal = part2.substring(0,5);
+		String locode = part2.substring(part2.length() - 2);
+		String city = part2.substring(6, part2.length() - 3);
+		int c_City_ID = getC_City_ID(city);
+		
+		MLocation location = new MLocation(Env.getCtx(), 0, get_TrxName());
```

Comment:
Thanks @Effecitech
This is not updating an address, but creating a new one.

If that's the intention, then the parameter must be named better "Create new Address" (instead of "Update Address")